src/main/java/frc/robot/util/TimeDelayedBoolean.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.util;
+
+import edu.wpi.first.wpilibj.Timer;
+
+/**
+ * This class contains a boolean value and a timer. It can set its boolean value and return whether the timer is within
+ * a set timeout. This returns true if the stored value is true and the timeout has expired.
+ */
+public class TimeDelayedBoolean {
```

Comment:
Isn't this just a Debouncer? It detects if a boolean is true for a certain amount of time.

Reply:
yeah ill delete it.